ProjectSettings/ProjectSettings.asset

```diff
@@ -756,11 +756,11 @@ PlayerSettings:
   apiCompatibilityLevel: 6
   activeInputHandler: 2
   windowsGamepadBackendHint: 0
-  cloudProjectId: 0a61489b-d67f-4618-a10d-fb549a98f782
+  cloudProjectId: 
   framebufferDepthMemorylessMode: 0
   qualitySettingsNames: []
-  projectName: Alien Arena 2024-01-20_14-41-47
-  organizationId: missing-rung-studios
+  projectName: 
+  organizationId: 
```

Comment:
Intentional change?

Reply:
This removes the unity devops part... it might be worth just killing these in a pr

Reply:
Oh okay, makes sense to me!